src/internal/common/dttm/dttm_test.go

```diff
@@ -58,6 +58,49 @@ func (suite *DTTMUnitSuite) TestParseTime() {
 	require.Error(t, err, clues.ToCore(err))
 }
 
+func (suite *DTTMUnitSuite) TestOrNow() {
+	var (
+		oneMinuteAgo = time.Now().Add(-1 * time.Minute)
+		inOneMinute  = time.Now().Add(1 * time.Minute)
+		inOneHour    = time.Now().Add(1 * time.Hour)
+		empty        = time.Time{}
```

Comment:
nit: we've generally not been explicitly setting to the zero value but just giving the type